src/api/local-storage.js

```diff
@@ -0,0 +1,86 @@
+class localStorageApi {
+ 
+  static createUser(newUser) {
+      localStorage.setItem("newUser", JSON.stringify(newUser));
+      var existingUsers = localStorageApi.getUsers();
+      existingUsers.push(newUser);
+      localStorage.setItem("allStoredUsers", JSON.stringify(existingUsers));
+      console.log("createUser() new current existing users = ", existingUsers);
+      return existingUsers;
+  }
+
+  static getUser(id) {
+      var existingUsers = localStorageApi.getUsers();
+      var found = existingUsers.find(function(allStoredUsers) {
```

Comment:
Maybe an arrow function here? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions

```
  var found = existingUsers.find(allStoredUsers => {
     return allStoredUsers.id === id;
  }
```

Or if you wanted to do some destructuring!

```
  var found = existingUsers.find({id} => {
     return id === userId; <-- you'd have to change the function param name to userId for this to work
  }
```

On destructuring: https://wesbos.com/destructuring-objects/